src/tree_schema.h

```diff
@@ -419,6 +419,10 @@ typedef enum {
 #define LYEXT_OPT_CONTENT    0x04    /**< content of lys_ext_instance_complex is copied from source (not dup, just memcpy). */
 /** @endcond */
 #define LYEXT_OPT_VALID      0x08    /**< needed to call calback for validation */
+#define LYEXT_OPT_SUBTREE    0x10    /**< The plugin needs to do validation on nodes in the subtree of the extended node
```

Comment:
This flag meaning is that the whole subtree should be validated but the name does not convey this. Maybe rename it to `LYEXT_OPT_VAL_SUBTREE` or something better?

Reply:
You're right.

At first, I wanted to name it `LYEXT_OPT_VALID_SUBTREE` (and its related `LYS_VALID_EXT_SUBTREE` schema flag) so they have the same prefix that  `LYEXT_OPT_VALID` and `LYS_VALID_EXT`.

However these names are quite long and it breaks source alignment. Would that be OK anyway?

Reply:
Always better than having vague names.

Reply:
I pushed the rename, please let me know if there are other changes required.